optuna/study/study.py

```diff
@@ -222,6 +225,8 @@ def get_trials(
         """Return all trials in the study.
 
         The returned trials are ordered by trial number.
+        -- seealso::
+            See :func:`~optuna.study.Study.trails` for related method.
```

Comment:
- Fix syntax: replace `--` with `::`
- Fix typo: trail -> trial 
- Replace `func` with `attr` 

```suggestion

        .. seealso::
            See :attr:`~optuna.study.Study.trials` for related method.
```